test-result-summary-client/src/Build/ReleaseSummary.jsx

```diff
@@ -0,0 +1,140 @@
+import React, { Component } from 'react';
+import ReactDOM from 'react-dom';
+import { getParams } from '../utils/query';
+import { Button, Tooltip, Card, Alert } from 'antd';
+import { CopyOutlined } from '@ant-design/icons';
+import TestBreadcrumb from './TestBreadcrumb';
+
+export default class ReleaseSummary extends Component{
+    state = {
+        body: "Generating Release Summary Report...",
+    }
+
+    async componentDidMount(){
+        await this.updateData();
+    }
+
+    async updateData(){
+        const { parentId } = getParams(this.props.location.search);
+        
+        //get build info
+        let fetchBuild = {};
+        fetchBuild = await fetch(`/api/getParents?id=${parentId}`, {
+            method: 'get'
+        });
+        const build = await fetchBuild.json();
+
+        //add build and test details to report
+        const report = `TRSS [link](https://trss.adoptopenjdk.net/buildDetail?parentId=${parentId}&testSummaryResult=failed&buildNameRegex=%5ETest) \n`;
```

Comment:
We have multiple TRSS servers for Adopt and Internal with different urls, could you use a variable (maybe exist already) to replace fixed url, so it will work with all TRSS servers, including local ones. Thanks.

Reply:
I have replaced the fixed origin URL with `window.location.origin` to use the base url that is calling the Release Summary functionality. Please let me know if that is what is required. Thank you!